client/components/payment-method-capability-status-pill/__tests__/payment-method-capability-status-pill.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import PaymentMethodCapabilityStatusPill from '..';
+import { useGetCapabilities } from 'wcstripe/data/account';
+import UpeToggleContext from 'wcstripe/settings/upe-toggle/context';
+
+jest.mock( 'wcstripe/data/account', () => ( {
+	useGetCapabilities: jest.fn(),
+} ) );
+
+describe( 'PaymentMethodCapabilityStatusPill', () => {
+	beforeEach( () => {
+		useGetCapabilities.mockReturnValue( {} );
+	} );
+
+	it( 'should render the "Pending activation" text', () => {
+		useGetCapabilities.mockReturnValue( {
+			giropay_payments: 'pending',
+			card_payments: 'active',
```

Comment:
What does `card_payments` do in this test? Is it so that it resembles closer to the actual `useGetCapabilities()` return values?